core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -693,13 +693,15 @@ public OptimizelyJSON getFeatureVariableJSON(@Nonnull String featureKey,
                     featureDecision.variation.getVariableIdToFeatureVariableUsageInstanceMap().get(variable.getId());
                 if (featureVariableUsageInstance != null) {
                     variableValue = featureVariableUsageInstance.getValue();
+                    logger.info("Got variable value \"{}\" for variable \"{}\" of feature flag \"{}\".", variableValue, variableKey, featureKey);
                 } else {
                     variableValue = variable.getDefaultValue();
+                    logger.info("Variable \"{}\" value is not defined. Returning default value \"{}\".", variableKey, variableValue);
```

Comment:
Slight correction needed in the first sentence. It will be awkward to put apostrophe. May be say `Value is not defined for variable "{}"`